lib/tasks/bulk.rake

```diff
@@ -325,4 +327,45 @@ namespace :bulk do
     attrs = { append_collection_ids: Valkyrie::ID.new(append_coll), skip_validation: true }
     BulkEditService.perform(collection_id: Valkyrie::ID.new(coll), attributes: attrs, metadata_adapter: Valkyrie::MetadataAdapter.find(:indexing_persister), logger: logger)
   end
+
+  desc "Ingest resources from a csv file"
+  task from_csv: :environment do
+    basedir = ENV["BASEDIR"]
+    coll = ENV["COLL"]
+    csvfile = ENV["CSV"]
+
+    abort "usage: COLL=collection_id BASEDIR=directory CSV=csvfile rake bulk:from_csv" unless coll && basedir
+    abort "no such file #{csvfile}" unless File.file?(csvfile)
+
+    user = User.find_by_user_key(ENV["USER"]) if ENV["USER"]
+    user = User.all.select(&:admin?).first unless user
```

Comment:
There's a rubocop error about this not being used — and I'm looking throughout the file, and I don't see the `user` variable being used  in most of the places it's defined. Maybe we need to be adding `depositor: user` to most of the properties to make sure that gets in?

Reply:
I copy/pasted the base of this task from somewhere else and this came along.  Removing it for now.